### ISSUE TITLE
[BUG] Chrome warning

### ISSUE THREAD

**Issue by BabsLabs** [BabsLabs/simple-sumo-backend#19]
**Describe the bug**
There is a new warning in chrome saying that the cookies must have proper settings.
```
(index):1 A cookie associated with a cross-site resource at http://babslabs-simple-sumo-backend.herokuapp.com/ was set without the `SameSite` attribute. It has been blocked, as Chrome now only delivers cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
```

**Possible place to start:**
https://stackoverflow.com/questions/58270663/samesite-warning-chrome-77

**Comment by BabsLabs** [comment]
According to my research this will be addressed in Rails 6.1 which will release in a few weeks. See [this official Rails link](https://github.com/rails/rails/issues/39190) for more info.

**Comment by BabsLabs** [comment]
According to my research this will be addressed in Rails 6.1 which will release in a few weeks. See this official Rails link: `githubbb.com/rails/rails/issues/39190` for more info.

I can either wait to Rails 6.1 to address or I can use one of the temporary patches such as the rails_same_site_cookie gem or other suggestions on this Rails Github post: `githubbb.com/rails/rails/pull/28297`
